dishdive/app/restaurants/[business_id]/page.tsx

```diff
@@ -0,0 +1,17 @@
+"use client";
+
+import DetailPage from "@/components/DetailPage/DetailPage";
+
+interface RestaurantPageProps {
+  params: {
+    business_id: string;
```

Comment:
update the dir name, so we can do 
```suggestion
    businessId: string;
```